stats/view/worker_commands.go

```diff
@@ -41,7 +41,12 @@ type getViewByNameResp struct {
 }
 
 func (cmd *getViewByNameReq) handleCommand(w *worker) {
-	cmd.c <- &getViewByNameResp{w.views[cmd.name].view}
```

Comment:
I think this new change now hides an error that should never occur i.e. a nil viewInternal.